PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingView.kt

```diff
@@ -16,16 +17,18 @@ import com.paypal.messages.config.message.PayPalMessageViewStateCallbacks
  * Note: **This module is in beta. It's public API may change or be removed in future releases.**
  * @property braintreeClient a {@link BraintreeClient}
```

Comment:
Would we want to update the kdoc with the `context` param?

Reply:
Good catch, updated: https://github.com/braintree/braintree_android/pull/915/commits/35b1b76d31195efd27639d320fbaa58265a52f7c